File: 1_Programing_Language/C/PRACTICE/Practice_13.c
#include<stdio.h>
#define CUBE(X) X*X*X
int main()
{
    int num;
    for(num=0;num<999;num++)
        if(num==CUBE((num%10))+CUBE(((num%100)/10))+CUBE((num/100)))
            printf("%d ",num);
    return 0;
}

File: 1_Programing_Language/Python/Practice/Practice_13.py
"""
定义一个函数，sun_numebrs，可以接收的 任意多个整数
功能要求：将传递的 所有数字累加 并且返回累加结果
"""

def sum_numbers(*args):

    num = 0
    for n in args:
        num += n

    return num

print(sum_numbers(1, 2, 3))


